unity/PitayaExample/LibPitaya.nuspec

```diff
@@ -8,7 +8,9 @@
     <projectUrl>https://github.com/topfreegames/libpitaya</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>C# client for the pitaya server</description>
-    <releaseNotes>Fix compatibility with Unity 2019.3</releaseNotes>
+    <releaseNotes>Fix crash in Unity 2019.3 when running in Linux
```

Comment:
Didn't you forget the version as well here?

Reply:
Yes! I pushed a new commit fixing it.